pkg/kubernetes/README.md

```diff
@@ -0,0 +1,29 @@
+# Kubernetes Code Copy Area
+
+This directory is the copy of original types from the main 
+[kubernetes/kubernetes repo](https://github.com/kubernetes/kubernetes) with 
+maintaining original packages structure. The purpose of this package is to
+avoid depending on the entire main repo and copy only necessary types as needed.
```

Comment:
It might be good to be very explicit that the files have been gutted, for example componentconfig/helpers.go has a bunch of stuff removed from it. It wasn't obvious to me from the wording. If for example there are some changes needed in the future and somebody looks at the file, they might get confused about what all they might need to add.

Reply:
Yeah initially I wanted to copy entire files, but in some places like `helpers.go` it seemed unnecessary as the unused code was importing other stuff we don't need.